src/core/redux/actions/index.js

```diff
@@ -2,6 +2,17 @@
  * @flow
  */
 
+const RESET_REQUEST_STATE :'RESET_REQUEST_STATE' = 'RESET_REQUEST_STATE';
+type ResetRequestStateAction = {|
+  requestStateAction :string[];
+  type :typeof RESET_REQUEST_STATE;
+|};
+
+const resetRequestState = (requestStateAction :string[]) :ResetRequestStateAction => ({
+  requestStateAction,
+  type: RESET_REQUEST_STATE,
+});
```

Comment:
I don't think `resetRequestState` (singular) is necessary because `resetRequestStates` (plural) does the same thing.

Reply:
yeah, at some point I was storing the request state in the `[ACTION, EKID]`, so passing multiple actions wasn't working for that. I needed a path. I'm not doing that anymore, so I'll remove it.

Reply:
I've gone through the exact same thing, where I have to fire off the same action multiple times and was trying to figure out how to keep track of it. One idea was to store as `[ACTION, ACTION_ID]`, which is similar to what you wrote. By the way, why did you use `EKID`?

In the end, this approached didn't pan out. I was hoping to be able to significantly improve the developer experience but it wasn't any better.